dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts

```diff
@@ -26,35 +26,70 @@ export const FormFieldProps = {
     type: Boolean,
     default: false,
   },
+  innerClass: {
+    type: String,
+    default: "short",
+  },
 };
 
-export const YesNoFormFieldProps = Object.assign(deepCopyObject(FormFieldProps), {
+export const YesNoFormFieldProps = Object.assign(deepCopyObject(BaseFormFieldProps), {
   certificateRequiredIfYes: {
     type: Boolean,
     default: false,
   },
 }) as Readonly<ComponentPropsOptions>;
-export const FormFieldPropsWithPlaceholder = Object.assign(deepCopyObject(FormFieldProps), {
+
+export const FormFieldPropsWithPlaceholder = Object.assign(deepCopyObject(BaseFormFieldProps), {
   placeholder: {
     type: String,
     default: "",
   },
 }) as Readonly<ComponentPropsOptions>;
+
 export const DateFormFieldProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
   todayAsMax: {
     type: Boolean,
     default: false,
   },
 }) as Readonly<ComponentPropsOptions>;
+
 export const DropdownOptionFormFieldProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
   options: {
     type: Array as () => Array<DropdownOption> | undefined,
     required: true,
   },
 }) as Readonly<ComponentPropsOptions>;
+
 export const OptionsFormFieldProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
   options: {
-    type: Array as () => Array<typeof Option> | undefined,
+    type: Object,
+    required: true,
+  },
+}) as Readonly<ComponentPropsOptions>;
+
+export const MultiSelectFormProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
+  options: {
+    type: Array as () => Array<DropdownOption>,
     required: true,
   },
+  optionValue: {
+    type: [String, Boolean] as PropType<string | boolean>,
+    default: "value",
+  },
+  optionLabel: {
+    type: [String, Boolean] as PropType<string | boolean>,
```

Comment:
These types are `string |Function` according to primevue.      Why have you implemented them this way?

Reply:
changed to type: [String, Function],

Reply:
👍